spec/deployer/base_spec.rb

```diff
@@ -139,10 +139,15 @@
   end
 
   context "#rm_command" do
+    before do
+      @test_dir = "/tmp/wordmove.delete"
+      FileUtils.mkdir(@test_dir)
+      FileUtils.touch(File.join(@test_dir, "my dump.sql"))
+    end
     let(:deployer) { described_class.new(:dummy_env) }
 
     it "creates a valid shell rm command" do
-      expect(deployer.send(:rm_command, "/var/my dump.sql")).to eq("rm /var/my\\ dump.sql")
+      expect(deployer.send(:rm_command, File.join(@test_dir, "my dump.sql"))).to eq(1)
```

Comment:
How integer 1 is considered "true" here? In a shell context 1 is an error exit status. In a method call context if the method returns true as boolean I'd write an expectation like `.to be true`. Am I missing something?